### DISCUSSION TITLE
How to sync geany setting of Windows to Linux one

### DISCUSSION THREAD

**Discussion by abdulbadii** [geany/geany#3842] [Q&A]
How do we synchronize geany setting of Windows' geany to Linux one i.e. Windows' geany setting follow the Linux'
And such for configurable components else too, (clearly point out step by step)
thanks before.

**Comment by elextr** [comment]
You can copy the whole configuration directory back and forwards but the experience might not be totally acceptable since:

1. session files, recent files, and other paths are likely to be in different locations
2. tools settings may not be the same between Windows and Linux
3. some settings may not exist in both, eg Windows does not have VTE.

Perhaps explain why you think you want them synchronised?
